config/routes.rb

```diff
@@ -46,4 +46,7 @@
   get 'dataset/:dataset_uuid/:name/datafile/:datafile_uuid/preview', to: 'previews#show', as: 'datafile_preview'
 
   get 'acknowledge', to: 'messages#acknowledge'
+
+  # Route everything else to CKAN
+  get '*path' => redirect('http://ckan.publishing.service.gov.uk/')
```

Comment:
This app also runs in staging, so is it possible to route requests on staging to ckan.staging.publishing.service.gov.uk?  There are environment configurations in this app, so it may be possible to define the redirect URL in there.

Also, I think we should route to https by default.